packages/excalidraw/components/App.tsx

```diff
@@ -1462,6 +1462,7 @@ class App extends React.Component<AppProps, AppState> {
       ) &&
       (this.state.selectionElement ||
         this.state.newElement ||
+        this.state.selectedElementsAreBeingDragged ||
```

Comment:
parity with prod: we need to disable UI pointer events when dragging elements over UI

Reply:
let's write a test for this